magento/magento2
# Fix the error that is wrong link title of a downloadable product when enabling "Use Default Value"

### Description (*)

Fix the error that is wrong link title of a downloadable product when enabling "Use Default Value"

### Related Pull Requests

N/A

### Fixed Issues (if relevant)


#27169 - Not able to update value with "use default checkbox" for Downloadable Product

### Manual testing scenarios (*)

1. Create a new downloadable product with a link
2. Choose scope at any store view (https://prnt.sc/rgpgs4)
3. In "Downloadable Information" section, uncheck "Use Default Value" box under link title and change link title then saving product
4. At the same product and scope, enable (check) "Use Default Value" box under link title then saving product

### Questions or comments


### Contribution checklist (*)
